bucket/cpu-z.json

```diff
@@ -36,7 +36,8 @@
         }
     },
     "persist": "cpuz.ini",
+    "checkver": "Version ([\\d.]+) for windows",
     "autoupdate": {
-        "url": "http://download.cpuid.com/cpu-z/cpu-z_$version-en.zip"
+        "url": "https://download.cpuid.com/cpu-z/cpu-z_$version-en.zip"
     }
 }
```

Comment:
I believe we need a new line at the end of file

Reply:
There aren't any new brackets or errors on preview, shouldn't be needed normally in that case - or do you just mean an empty line?

Reply:
Empty line

checkver.ps1 adds one to the end of file.

But I noticed a lot of files don't have them.

Let me check with /verify once

Reply:
Needs new line @RavenMacDaddy by the looks of it

Reply:
Looking at both commits in edit mode there seems to be an empty line at the end, does there need to be two?